### PR TITLE
Feature: Updated Slack

### PR DESCRIPTION
**Pull request by krak3n**
This PR updates the Slack payload to produce an output similar too:

Also some minor PEP8 niggles.

### CODE REVIEW

File: fmslack/cli.py [hunk 1]
```diff
@@ -166,6 +159,7 @@ def query_api(api_url, uri):
     except ValueError:
         return None
 
+
```
Comment (krak3n) [inline fmslack/cli.py:162]: Always 2 lines between function definitions.

File: fmslack/cli.py [hunk 2]
```diff
@@ -85,9 +85,9 @@ def slack(redis_uri, redis_channel, slack_webhook_url, api_url, log_level):
                 logger.info('Event: PLAY')
                 track = query_api(api_url, data['uri'])
                 if track is not None:
-                    logger.debug(
-                        'API returned track data for {0}'.format(track['spotify_uri']))
-                    slack = slack_post(
```
Comment (krak3n) [inline fmslack/cli.py:90]: `slack` was defined but never used, its generally bad practice to define variables and not use them :smile:

File: fmslack/cli.py [hunk 3]
```diff
@@ -113,28 +113,21 @@ def slack_post(slack_webhook_url, name, artist, album, image):
...
+        "username": "thisissoon.fm",
+        "icon_url": image
     }
-    headers = {'content-type': 'application/json'}
-    r = requests.post(slack_webhook_url, data=json.dumps(payload), headers=headers)
+
+    r = requests.post(slack_webhook_url, data=json.dumps(payload), headers={
+        'content-type': 'application/json'
+    })
+
```
Comment (krak3n) [inline fmslack/cli.py:130]: I'm just being picky here but I think this looks neater.

Comment (krak3n) [inline fmslack/cli.py:130]: Alternative:

``` python
   response = requests.post(
        slack_webhook_url,
        data=json.dumps(payload),
        headers={
            'content-type': 'application/json'
        })
```

**Comment by jamesjwarren** [comment]
Nice, thanks for the feedback :smiley:

File: fmslack/cli.py (hunk 3 above)
Comment (krak3n) [inline fmslack/cli.py:130]: I've not fixed it but in general we don't like single character named variables in python. So `r = request.post...` should really be `response = request.post...`

**Comment by krak3n** [comment]
@jamesjwarren if you think this is cool I'll get it deployed.

**Comment by jamesjwarren** [comment]
I'm happy, looks great!

**Pull request merged**